fintech-examples/fintech-ui/src/app/bank/common/transaction-card/transaction-card.component.html

```diff
@@ -10,7 +10,7 @@
         }}</span>
       </div>
       <div class="right">
-        <span class="ltx-amount-number">{{ transaction?.transactionAmount?.amount }}</span>
+        <span class="ltx-amount-number{{transaction?.transactionAmount?.amount<0?'-red':''}}">{{ transaction?.transactionAmount?.amount }}</span>
```

Comment:
Please use NgClass directive in the line 12.
`[ngClass]="{'text-danger': transaction?.transactionAmount?.amount < 0}"`